streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBSessionStore.java

```diff
@@ -41,27 +40,26 @@
                         final Serde<AGG> aggSerde) {
         super(bytesStore);
         this.keySerde = keySerde;
-        this.bytesStore = bytesStore;
         this.aggSerde = aggSerde;
     }
 
     @Override
     @SuppressWarnings("unchecked")
     public void init(final ProcessorContext context, final StateStore root) {
-        final String storeName = bytesStore.name();
+        final String storeName = name();
```

Comment:
Why not `wrappedStore().name()` of ofter getting access to the variable, just `wrapped.name()` ?

Reply:
`name()` is provided by the super class, since it is a `StateStore` method. I'd consider `super.name()`, but it seemed better to elide the unnecessary `super.` the same way we do for `this.`.

As with the prior discussion about `super.init()`, it would be wrong to circumvent the superclass's method and make assumptions about its implementation.